lib/jsonld-signatures.js

```diff
@@ -364,6 +357,32 @@ function _getSupportedAlgorithms() {
   return Object.keys(api.suites).filter(s => s !== 'LinkedDataSignature');
 }
 
+function _isSupportedProofPurpose(purpose) {
+  if(!purpose) {
+    // TODO: Enable error in the next major release
```

Comment:
We should connect this with an issue, otherwise it will linger here unnoticed.

Reply:
todo pending results of conversation in issue: https://github.com/digitalbazaar/jsonld-signatures/issues/41